vbr/vbrCore/functions/el_ModUnrlx_dTdP_f.m

```diff
@@ -28,11 +28,25 @@
   dT = (VBR.in.SV.T_K-T_K_ref);
   dP = (VBR.in.SV.P_GPa*1e9 - P_Pa_ref);
 
-  % calculate shear modulus at T,P of interest
-  Gu_TP = calc_Gu(Gu_0,dT,dP,dG_dT0,dG_dP0);
+  if isfield(VBR.in.elastic,'Gu_TP') && isfield(VBR.in.elastic,'Ku_TP')
+    % Load unrelaxed shear and bulk moduli (at T,P of interest)
+    Gu_TP = VBR.in.elastic.Gu_TP; % Pa
+    Ku_TP = VBR.in.elastic.Ku_TP; % Pa
```

Comment:
let's also overwrite `VBR.out.elastic.Gu_0` so that it matches 
```suggestion
    Ku_TP = VBR.in.elastic.Ku_TP; % Pa
    VBR.out.elastic.Gu_0 = Gu_TP; 
```

Reply:
hmm. maybe not actually. maybe we should set  VBR.out.elastic.Gu_0  to 0 instead. or delete the field. let me think about that...